{
  "affected_release": null,
  "package_state": [
    {
      "product_name": "Red Hat Enterprise Linux 5",
      "fix_state": "Will not fix",
      "package_name": "gedit",
      "cpe": "cpe:/o:redhat:enterprise_linux:5"
    }
  ],
  "threat_severity": "Low",
  "public_date": "2008-08-06T00:00:00Z",
  "bugzilla": {
    "description": "CVE-2009-0314 gedit: untrusted python modules search path",
    "id": "481556",
    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=481556"
  },
  "cvss": {
    "cvss_base_score": "",
    "cvss_scoring_vector": "",
    "status": ""
  },
  "cvss3": {
    "cvss3_base_score": "",
    "cvss3_scoring_vector": "",
    "status": ""
  },
  "iava": "",
  "cwe": "",
  "statement": "This issue does not affect gedit as shipped in Red Hat Enterprise Linux 3 and 4. It does affect gedit in Red Hat Enterprise Linux 5.\nRed Hat Product Security has rated this issue as having Low security impact. This issue is not currently planned to be addressed in future updates. For additional information, refer to the Issue Severity Classification: https://access.redhat.com/security/updates/classification/.",
  "acknowledgement": "",
  "name": "CVE-2009-0314",
  "document_distribution": "",
  "details": [
    "Untrusted search path vulnerability in the Python module in gedit allows local users to execute arbitrary code via a Trojan horse Python file in the current working directory, related to a vulnerability in the PySys_SetArgv function (CVE-2008-5983)."
  ],
  "references": [
    "https://www.cve.org/CVERecord?id=CVE-2009-0314\nhttps://nvd.nist.gov/vuln/detail/CVE-2009-0314"
  ]
}